frontend/src/app/prereq-tree/node/node.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Jw what is this whole file doing?

Reply:
I copied it from the source project, and it looks it is used for testing although we don't explicitly do any tests. Every `.ts` file in angular is associated with a file for testing: `.spec.ts` 

More info: 
https://stackoverflow.com/a/37502922
https://angular.io/guide/testing